runtime/src/main/scala/akka/grpc/internal/GrpcRequestHelpers.scala

```diff
@@ -37,6 +37,7 @@ object GrpcRequestHelpers {
     HttpRequest(
       uri = uri,
       method = HttpMethods.POST,
+      // FIXME how can client exclude gzip?
```

Comment:
how should we support this? don't include the `Message-Accept-Encoding` header by default and only include it via the `headers` parameter? What will the end user api be for specifying the header? The low level metadata api or additional CallOption?

Reply:
created issue https://github.com/akka/akka-grpc/issues/1382 for this